diesel_derives/tests/schema.rs

```diff
@@ -11,5 +12,6 @@ table! {
         id -> Integer,
         name -> Text,
         hair_color -> Nullable<Text>,
+        r#type -> Nullable<Text>,
```

Comment:
Note that the CLI currently would generate `type_` in such scenarios:
```rust
table! {
	things(id) {
		id -> Int4,
		#[sql_name = "type"]
		type_ -> Int4,
	}
}
```

(I understand that this only adds support and does not change behavior of the CLI)

Reply:
Should diesel_cli be changed to emit `r#type` in this case now?

Reply:
I think adding the underscore is typically more practical.